### PR TITLE
Fix typos in resources/payment.py

### LINKED ISSUE #95: Unexpected parameter in docstring of fetching transfers for 'payment_id'
## [Fetch Transfers for a Payment](https://razorpay.com/docs/api/route/#fetch-transfers-for-a-payment)

This function doesn't require any `Amount` parameter.

https://github.com/razorpay/razorpay-python/blob/50cfc8269ea498751671770a7eb506f7eb91cbb9/razorpay/resources/payment.py#L79-L87

### PR DESCRIPTION
**Pull request by Vibhu-Agarwal**
Resolves #95 

... and other minor typos

### CODE REVIEW

File: razorpay/resources/payment.py [hunk 1]
```diff
@@ -83,7 +83,6 @@ def transfers(self, payment_id, data={}, **kwargs):
 
         Args:
             payment_id : Id for which payment object has to be refunded
```
Comment (harman28) [inline razorpay/resources/payment.py:85]: Could this line be updated as well?

Comment (Vibhu-Agarwal) [reply razorpay/resources/payment.py]: Fixed :+1:

**Comment by Vibhu-Agarwal** [comment]
@harman28 Made the requested changes and fixed few more (missed out) typos :)

**Comment by Vibhu-Agarwal** [comment]
@captn3m0

**Pull request merged**